cfme/infrastructure/deployment_roles.py

```diff
@@ -71,59 +264,45 @@ def step(self):
         except NoSuchElementException:
             nav_select('Clusters / Deployment Roles')
 
-    def am_i_here(self):
-        def match(option):
-            return match_page(title=option, summary=option)
-
-        return match('Clusters / Deployment Roles') or match('Deployment Roles')
+        if 'Grid View' not in self.view.toolbar.view_selector.selected:
```

Comment:
**REQUIRED**
When you use entities you don't need to reset the view to specific one like "Grid" or "Table". This stuff is not needed.